management/server/http/users_handler_test.go

```diff
@@ -340,6 +351,69 @@ func TestCreateUser(t *testing.T) {
 	}
 }
 
+func TestInviteUser(t *testing.T) {
+	notFoundUserToInvite := api.UserInviteRequest{
+		Email: notFoundUserEmail,
+	}
+
+	notFoundUserReqPayload, err := json.Marshal(notFoundUserToInvite)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	userToInvite := api.UserInviteRequest{
+		Email: "email",
+	}
+
+	userToInviteReqPayload, err := json.Marshal(userToInvite)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tt := []struct {
+		name           string
+		expectedStatus int
+		requestType    string
+		requestPath    string
+		requestBody    io.Reader
+		expectedResult []*server.User
+	}{
+		{
+			name:           "Invite User",
+			requestType:    http.MethodPost,
+			requestPath:    "/api/users/" + existingUserID + "/invite",
+			expectedStatus: http.StatusOK,
+			requestBody:    bytes.NewBuffer(userToInviteReqPayload),
+		},
+		{
+			name:           "Invite User with Existing User",
```

Comment:
The test case names should represent/describe whats tested within this case so would be found user email and not found user email